openreview/venue_request/process/deployProcess.py

```diff
@@ -118,23 +118,30 @@ def process(client, note, invitation):
                     'required': True,
                     'order': 2
                 },
+                'invitee_reduced_load': {
+                    'description': 'Please enter a comma separated value of the reduced load options if the invitee rejected the invitation. (Only for reviewer role)',
```

Comment:
Maybe something like:
"Please enter a comma separated list of reduced load options. If an invitee rejects the reviewing invitation, they will be able to choose a reduced load from this list. (For reviewer role only)"

Might seem wordy but I think it's a bit more understandable.